packages/itwinui-react/package.json

```diff
@@ -99,20 +100,18 @@
     "@types/react-transition-group": "^4.4.10",
     "@typescript-eslint/eslint-plugin": "^5.60.0",
     "@typescript-eslint/parser": "^5.60.0",
+    "@vitest/coverage-v8": "^1.2.1",
     "concurrently": "^5.3.0",
     "eslint": "^8.43.0",
     "eslint-config-prettier": "^8.8.0",
     "eslint-plugin-require-extensions": "^0.1.3",
-    "jest": "^29.0.0",
```

Comment:
After doing a global search for "jest", I still found some occurances. Those will need to be handled too.